neo/Persistence/Memory/Helper.cs

```diff
@@ -0,0 +1,12 @@
+namespace Neo.Persistence.Memory
+{
+    internal static class Helper
+    {
+        private static readonly byte[] EmptyBytes = new byte[0];
+
+        public static byte[] EnsureNotNull(this byte[] source)
+        {
```

Comment:
This should be a generic helper for all Storages?

Reply:
It is currently used in `Memory` namespace only.

Reply:
But all of them should work equal with the keys, isn't it?
If a smart contract store a `null` key, we should return the same in all storages

Reply:
This `Helper` is used in `Memory` only. I'm not saying that this strategy is for `MemoryStore` only...